src/ContactsTabManagerInterface.php

```diff
@@ -69,12 +69,12 @@ public function getBlocks(ContactTabInterface $tab, UserInterface $contact = NUL
    *
    * @param \Drupal\contacts\Entity\ContactTabInterface $tab
    *   The tab.
-   * @param \Drupal\user\UserInterface $contact
+   * @param \Drupal\user\UserInterface|null $contact
    *   The contact.
    *
    * @return bool
    *   Whether the tab is valid.
    */
-  public function verifyTab(ContactTabInterface $tab, UserInterface $contact);
+  public function verifyTab(ContactTabInterface $tab, UserInterface $contact = NULL);
```

Comment:
This is making quite a lot of change, presumably due to not wanting to require validating for a contact for manage mode. Perhaps it's worth considering manage mode having a completely separate pipeline?